CONTRIBUTING.md

```diff
@@ -144,3 +144,42 @@ NOTE: There are three test apps in the repository, located in the `/demo` direct
         ```
 
 Thank you for your contribution!
+
+## Publishing new versions
```

Comment:
Add to ToC: https://github.com/NativeScript/nativescript-dev-webpack/blob/master/CONTRIBUTING.md#contributing.